### PR TITLE
Improve quickstart page to more involved example

### PR DESCRIPTION
**Pull request by Globegitter**
Since it would have been useful for me to have a more complex example I thought I would create one that is quite similar to one of our current use cases. It still doesn't show `ConnectionFields`, but maybe that would be a bit too much anyway for people who are completely new to graphql and/or relay.

### CODE REVIEW

**Comment by syrusakbary** [comment]
Thats a good advice! However a quickstart should need to have everything needed for "warming up" and "working" doing copy&paste.
If you could make the example work with just one file (not importing `somewhere`) and working I will love to merge it.
PS: Maybe doing a very simple `dict` memory storage solution will work :wink:

**Comment by Globegitter** [comment]
`user_id` is a field in the `User` Object. The reason we use separate `id` and `..._id` in our apps is so we can display our actual IDs. But happy to completely remove it and write as you suggested.

Otherwise addressed your comments :)

File: quickstart.md [hunk 1]
```diff
@@ -33,20 +33,101 @@ pip install graphene
...
+First we're going to define some GraphQL ObjectTypes and relay Nodes for our `Schema`:
 
 
 ```python
 import graphene
+from graphene import relay
+
+USER_DATA = [
+    {
+        user_id = '1'
```
Comment (syrusakbary) [inline quickstart.md]: This syntax is not python valid :( (have to be key: value)

**Comment by Globegitter** [comment]
Sorry about that - my head was clearly on kwargs.

File: quickstart.md [hunk 2]
```diff
@@ -33,20 +33,101 @@ pip install graphene
...
 class Query(graphene.ObjectType):
-    username = graphene.StringField(description='The username')
-
-   	def resolve_username(self, *args):
-   		return 'Hello World'
+        users = graphene.List(User,
+                            fullName=graphene.String(),
+                            sortKey=graphene.String(),
+                            sortDirection=graphene.String())
+        user = relay.NodeField(Users)
```
Comment (ptz0n) [inline quickstart.md]: I cannot find where you have defined the `Users` class.

Comment (jhgg) [inline quickstart.md]: Typo. Should be `User` I think.

**Comment by syrusakbary** [comment]
I have to see if everything works, and if is the case I will approve soon! @Globegitter

**Comment by ptz0n** [comment]
Please have a look in Python 2.7 as well. I see alot of syntax errors in the code right now.

**Comment by ptz0n** [comment]
@Globegitter is this really working for you? Output from [`test.py`](https://gist.github.com/ptz0n/465eca6fa2ab1161fbbd) is `None`.

Works if you change `sort_directon` to `sort_direction`. :+1:

File: quickstart.md [hunk 3]
```diff
@@ -33,20 +33,101 @@ pip install graphene
...
+    return user
+
+def fetch_users_from_database(args):
+    full_name = args.get('fullName', None)
+    if full_name:
+        full_name = full_name.upper()
+    sort_key = args.get('sortKey', None)
+    sort_direction = args.get('sortDirection', None)
+    reversed = False
+    if sort_directon and sort_directon == 'DESC':
```
Comment (ptz0n) [inline quickstart.md]: This one, should be `sort_direction`.

**Comment by Globegitter** [comment]
@ptz0n Yeah was working for me, but rather than copy&pasting the fixes I just typed them. And well seemed to have been a bit blind. Anyway should be good now.

**Comment by ptz0n** [comment]
Would be nice if you run `flake8` on the new example code.

**Comment by Globegitter** [comment]
Did just that @ptz0n :)

**Comment by adamcharnock** [comment]
Just thought I'd drop a note here to say that I plan to do a lot of work on the Django quickstart guide as part of PR #60.

**Comment by syrusakbary** [comment]
Closing this issue as the Quickstart guide docs are now more detailed now.
Please reopen if you feel that something have to be changed.

**Pull request closed without merging**